[tests-only] removed the search test from the expected failures

ScharfViktor opened the pull request:
related #4485 and https://github.com/owncloud/core/pull/40330

phil-davis commented:
I merged https://github.com/owncloud/core/pull/40330 and rebased here and updated the core commit id.
Let's see what CI thinks.

ScharfViktor commented:
> I merged [owncloud/core#40330](https://github.com/owncloud/core/pull/40330) and rebased here and updated the core commit id. Let's see what CI thinks.

Thank you Phil. CI is green. It looks like everything is fine.
